src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs

```diff
@@ -74,7 +74,7 @@ public GlobbingUrlBuilder(IFileProvider fileProvider, IMemoryCache cache, PathSt
         /// <param name="includePattern">The file globbing include pattern.</param>
         /// <param name="excludePattern">The file globbing exclude pattern.</param>
         /// <returns>The list of URLs</returns>
-        public virtual IEnumerable<string> BuildUrlList(string staticUrl, string includePattern, string excludePattern)
+        public virtual ICollection<string> BuildUrlList(string staticUrl, string includePattern, string excludePattern)
```

Comment:
Only reason going `ICollection<string>` here is to avoid `.Any()`. This is technically an `Internal` type though so I thought it was warranted.